R/FilteredDataset.R

```diff
@@ -322,17 +311,59 @@ FilteredDataset <- R6::R6Class( # nolint
             }
           )
 
-          private$observers[[session$ns("get_filter_state")]] <- observeEvent(self$get_filter_state(), {
+          is_filter_collapsible <- reactive({
+            length(self$get_filter_state()) != 0
```

Comment:
Let's create a `filter_count <- reactive()` and  remove `FilteredDataset$get_filter_count()`. 
If you remove `$get_filter_count()` ocurence here you'll be able to remove `get_filter_count` from all the classes.
 
```suggestion
            length(filter_count()) != 0
```